clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/LeaderFollowerPartitionStateModel.java

```diff
@@ -104,7 +104,7 @@ public void onBecomeStandbyFromOffline(Message message, NotificationContext cont
       try {
         long startTimeForSettingUpNewStorePartitionInNs = System.nanoTime();
         setupNewStorePartition();
-        heartbeatMonitoringService.addFollowerLagMonitor(store.getVersion(version).get(), getPartition());
+        store.getVersion(version).ifPresent(v -> heartbeatMonitoringService.addFollowerLagMonitor(v, getPartition()));
```

Comment:
We need this to fail more loudly, or, in the absence of this still set up a lag monitor.

Our goal is to set all alerting on this metric.  If we randomly aren't setting up the alert, then it won't be very useful.

Reply:
I agree. For now I can add a log statement but we should "understand and fix why version metadata is missing."

Reply:
Added configurable wait and log statement